internal/restapi/providers/v1/helper.go

```diff
@@ -0,0 +1,20 @@
+package v1
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func GetProviderNameFromRequest(r *http.Request) string {
```

Comment:
Can we have some tests for these 2 functions please!

Reply:
Thank you for the suggestion! I've added tests for these two functions.